### ISSUE TITLE
Set Parent Thumbnails

### ISSUE THREAD

**Issue by ShanaLMoore** [notch8/utk-hyku#304]
# Summary
UTK has noticed that parent works don't have thumbnails, and they aren't showing up on the home page, collections page, or works page.

The issue isn’t that they don’t have thumbnails, it’s just that they are pointing to filesets that didn’t produce them as derivatives. I think this is because they don’t have the [rdf_type value](https://github.com/scientist-softserv/utk-hyku/blob/main/app/jobs/create_derivatives_job_decorator.rb#L22) required for the [CreateDerivativeJob](https://github.com/scientist-softserv/utk-hyku/blob/main/app/jobs/create_derivatives_job_decorator.rb) to run, per this. 

rdf_type is a property that is only available to the Attachment model, which is an extension of a file_set. 

Ref [PR](https://github.com/scientist-softserv/utk-hyku/commit/d76f54fdf3257c337dd14ef453451c82c50ad3bc)

# Acceptance Criteria

- [ ] Parent thumbnails show on the home page
- [ ] Parent thumbnails show on the collections page
- [ ] Parent thumbnails show on the work page

# Screenshots or Video

# Testing Instructions

# Notes

Confused how to fill this out? Check out the [playbook](https://playbook-staging.notch8.com/en/notch8/process/issues)

**Comment by ShanaLMoore** [comment]
Bringing this issue to the team sprint planning mtg because it make impact ingests (and it may need to get prioritized).

I wrote a script to unblock Amanda for local development, until this gets resolved properly. The script is inspired by [LV's set_parent_thumbnail rake task](https://github.com/scientist-softserv/louisville-hyku/blob/main/app/jobs/set_default_parent_thumbnail_job.rb).  She's working on thumbnail styling and was unable get the works to produce thumbnails.

**Comment by jillpe** [comment]
related to #7

**Comment by jillpe** [comment]
duplicate of #438
